src/main/java/skynet/RobotWarSimulation.java

```diff
@@ -0,0 +1,45 @@
+package skynet;
+
+import java.util.Random;
+
+
+/**
+ * Main simulation class for the Robot War.
+ * Simulates 100 days of part production and collection
+ * and determines which faction builds the most robots.
+ */
+public class RobotWarSimulation {
```

Comment:
No mechanism exists to properly terminate the threads after the simulation completes.